Case list icons

wpride opened the pull request:
Resolve case list media properly

<img width="1148" alt="screen shot 2016-06-06 at 4 11 08 pm" src="https://cloud.githubusercontent.com/assets/2293064/15836149/7662afb2-2c01-11e6-92a9-f4b8ea0017d0.png">

dannyroberts committed:
use hqDefine for users/js/roles.js

Commit:
226216-case: Unsubscribe from scheduled emails - updated functionality

Commit:
Remove unused get_scheduled_report_response parameter

Commit:
Fix rendering notification with no request provided

Commit:
Corrected translations, disable redirection to home page

Commit:
Fix - Changed uuid generator and adjusted base template

dimagimon commented on corehq/apps/cloudcare/static/cloudcare/js/formplayer/js/apps/menus/menu_view.js:
```diff
@@ -72,6 +77,16 @@ FormplayerFrontend.module("SessionNavigate.MenuList", function (MenuList, Formpl
+            this.styles = options.styles;
+        },
+
+        childViewOptions: function (model, index) {
```
'index' is defined but never used (no-unused-vars)

NoahCarnahan committed:
400 instead of 404

NoahCarnahan committed:
limit

dannyroberts committed:
Merge pull request #11949 from dimagi/add-username-to-500

add username to 500 emails

emord committed:
Merge pull request #11770 from dimagi/jsonfield-ilsgateway

upgrade jsonfield for ilsgateway

snopoke committed:
don't submit forms as the mobile worker

emord committed:
vellum:8b8ea8e3b9c81e151a71cd8467496b29470bfff8

snopoke committed:
update to exclude system forms unless unkown users is specified

esoergel committed:
Merge pull request #11801 from dimagi/remove-unused-imports-form_processor

remove unused imports from corehq/form_processor

calellowitz committed:
Merge pull request #11917 from dimagi/b3-template-cleanups

B3 template cleanups

czue committed:
Merge pull request #11952 from dimagi/temp-user

fix temp user filter

calellowitz committed:
using proper ga protocol to track domains

proteusvacuum committed:
Dont hang on circular references

Commit:
update staging.yaml

[ci skip]

orangejenny committed:
Merge pull request #11954 from dimagi/form-display

Form display tweaks

Commit:
Make location lookup more graceful in bulk user import

biyeun committed:
update staging.yaml

[ci skip]

biyeun committed:
update staging.yaml

[ci skip]

Commit:
update staging.yaml

[ci skip]

Commit:
Update submodules

Commit:
Merge remote-tracking branch 'origin/master'

Commit:
update staging.yaml

[ci skip]

Commit:
Update translations.

[ci skip]

Commit:
Merge remote-tracking branch 'origin/master'

Commit:
update deployment

Commit:
update staging.yaml

[ci skip]

dannyroberts committed:
respond to delete_couch_devicelogs PR feedback

- use iter_docs_with_retry
- assert doc_type
- set throttle_secs=2

Commit:
Merge pull request #11961 from dimagi/delete_couch_devicelogs

respond to delete_couch_devicelogs PR feedback

dannyroberts committed:
fix lint review’s comma-dangle

dannyroberts committed:
update staging.yaml

[ci skip]

dannyroberts committed:
update staging.yaml

[ci skip]

esoergel committed:
Remove min_version

I'm pretty sure this is never relevant.  `min_version` is only ever set
if the app_id passed in corresponds to a released app, so there can't be
a situation where the latest released app has an ealier version, since
the set of released app versions *contains* the passed in app.

If I knew set theory, that could probably be proved.

esoergel committed:
change app_id instead of confusing parent_id thing

esoergel committed:
Don't try so hard - remove duplicate code

esoergel committed:
Just use app, drop original_app thing

dannyroberts committed:
Merge pull request #11960 from dimagi/user-import-location-error

Make location lookup more lenient in bulk user import

dannyroberts committed:
replace Exports.ViewModels with hqDefine

dannyroberts committed:
indent export/js/models.js

dannyroberts committed:
replace Exports.Constants with hqDefine

dannyroberts committed:
indent export/js/const.js

dannyroberts committed:
replace Exports.Utils with hqDefine

dannyroberts committed:
remove exports/js/exports.js

dannyroberts committed:
indent export/js/utils.js

czue committed:
Merge pull request #11959 from dimagi/domain-tracking

using proper ga protocol to track domains

czue committed:
Merge pull request #11956 from lwyszomi/care-fix-6

Care: Fixed problem with missing columns

czue committed:
Merge pull request #11950 from dimagi/reopen-usercase

Re-open usercase after form is un-archived

czue committed:
Merge pull request #11947 from dimagi/exclude-system-forms

exclude system forms form submit history

czue committed:
Merge pull request #11943 from dimagi/sql-close-extensions

Auto close extensions in SQL domains

Commit:
Added Commcare version report

kaapstorm committed:
Use a list, not a generator

Commit:
dimagimon

snopoke committed:
fix select text

czue committed:
Merge pull request #11948 from dimagi/border

remove border from form XML view

snopoke committed:
don't bail on form with stub

czue committed:
delete "Ledgers By Location" Report

czue committed:
delete toggle

snopoke committed:
pass context through NotFilter

czue committed:
extract function to analytics module

snopoke committed:
lint [ci skip]

czue committed:
return a dict instead of a list of tuples

Commit:
Merge pull request #11468 from dimagi/hqDefine-users

use hqDefine for users/js/roles.js

czue committed:
add basic tests for get_ledger_values_for_case_as_of

millerdev committed:
Merge pull request #11967 from dimagi/import-app

don't bail on form with stub

Commit:
Fix edit sms languages page

Commit:
Remove unused kwarg

orangejenny committed:
Merge pull request #11966 from dimagi/form-data

fix select text

kaapstorm committed:
Calculate excluded forms client-side

esoergel committed:
Merge pull request #11968 from dimagi/ledgers-by-location

delete "Ledgers By Location" Report

Commit:
Merge pull request #11969 from dimagi/not-root-doc

pass context through NotFilter

Commit:
Update submodules

snopoke committed:
don't fail on report modules

Commit:
Revert prelogin submodule

snopoke committed:
handle user with no submissions

Commit:
Remove deleted report from menu

czue committed:
Merge pull request #11973 from dimagi/fix-reports-page

Remove deleted report from menu

calellowitz committed:
removing fullstory from apps base and adding to individual templates

dannyroberts committed:
change script order in export/spec/ko/mocha.html

proteusvacuum committed:
Add MultipleTaskDownload for monitoring groups

proteusvacuum committed:
Add has_file to download objects

This allows saved download objects to not neccessarily contain files

proteusvacuum committed:
Get status of group tasks

proteusvacuum committed:
Complete tasks if CELERY_ALWAYS_EAGER is False

proteusvacuum committed:
Remove old jQuery reference

This was getting downloaded whenever a file download was initiated

proteusvacuum committed:
Wrap errors in <ul>

https://docs.djangoproject.com/en/1.9/ref/templates/builtins/#unordered-list

proteusvacuum committed:
UI if there are errors in some tasks but not all

NoahCarnahan committed:
Merge pull request #11974 from dimagi/fullstory-2

removing fullstory from apps base and adding to individual templates

proteusvacuum committed:
split prime restore into multiple tasks

proteusvacuum committed:
Use queue_prime_restore in view

proteusvacuum committed:
Delete soil specific jquery [ci skip]

dannyroberts committed:
eslint

dannyroberts committed:
Merge pull request #11907 from dimagi/shadow_form_select

Shadow module form selection

dannyroberts committed:
Merge pull request #11970 from dimagi/fix-edit-langs

Fix edit sms language page

proteusvacuum committed:
Use proper content type for less file

Commit:
update spacing of surrounding elements

Commit:
update payment option text

Commit:
remove dollar sign from tax rate

biyeun committed:
Merge pull request #11978 from dimagi/multiselect-less

Use proper content type for less file

Commit:
small refactor

czue committed:
fix prune_elastic_indices to correctly identify existing indices

snopoke committed:
store parsed versions in ES

snopoke committed:
Merge pull request #11981 from dimagi/prune-fix

fix prune_elastic_indices to correctly identify existing indices

kaapstorm committed:
Merge pull request #11939 from dimagi/ucr-export-limit

Limit UCR excel export size

snopoke committed:
store location in ES

snopoke committed:
rename case and form indexes

snopoke committed:
tests

snopoke committed:
use index name for pillow unique ID

dannyroberts committed:
change script order in corehq/apps/export/templates/export/customize_export_new.html

orangejenny committed:
Merge pull request #11972 from dimagi/unknown

handle user with no submissions

snopoke committed:
move under if meta

snopoke committed:
revert case index name change

dannyroberts committed:
fix bad constructor call

snopoke committed:
revert report case index name change

snopoke committed:
move pillow tests out of reports app

snopoke committed:
update test

snopoke committed:
use standard way of clearing default filters

snopoke committed:
use multi-topic checkpoint

snopoke committed:
rename

snopoke committed:
remove old imports

NoahCarnahan committed:
fix set operation

calellowitz committed:
update staging.yaml

[ci skip]

proteusvacuum committed:
Copy docs into correct dbs

snopoke committed:
Merge pull request #11975 from dimagi/multiple-task-download

Split prime restore into multiple tasks

NoahCarnahan committed:
Merge pull request #11964 from dimagi/hqDefine-exports

Use hqDefine in new export code

Commit:
remove unnecessary join()

snopoke committed:
Merge pull request #11983 from dimagi/unknown-user

Unknown user pillow test fixes

snopoke committed:
Merge pull request #11971 from dimagi/cc-config

don't fail on report modules

NoahCarnahan committed:
Merge pull request #11982 from dimagi/versions-in-es

store parsed versions and location in ES

esoergel committed:
Merge pull request #11799 from dimagi/remove-unused-imports-cleanup

remove unused imports from corehq/apps/cleanup

esoergel committed:
Merge pull request #11803 from dimagi/remove-unused-imports-sql_accessors

remove unused imports from corehq/sql_accessors/

esoergel committed:
Add param to include multimedia in ccz download

NoahCarnahan committed:
Allow floats and other numbers

Commit:
accept 0 as being valid field value

Commit:
use === instead of == per dimagimon

proteusvacuum committed:
stop circular import

dannyroberts committed:
sort docs in ptop_fast_reindex_cases

so that the most recent get reindexed first

Commit:
use !==

dannyroberts committed:
add ability to sort docs in ptop_fast_reindex_fast

by including docs but only using them in the sort key

Commit:
Merge pull request #11984 from dimagi/set-bug

Fix bug in contracted partners subscription form

emord committed:
Merge pull request #11985 from dimagi/copy-domain-multi-db

Copy docs into correct dbs with copy_domain

Commit:
remove obsolete hash_compat.py

Commit:
replace django.forms.util with django.forms.utils

NoahCarnahan committed:
Fix modal removal

esoergel committed:
Merge pull request #11979 from dimagi/invoice-text-changes

Invoice text changes

Commit:
ensure that endpoints are updated for j2me

Commit:
update staging.yaml

[ci skip]

Commit:
Merge pull request #11988 from dimagi/ccz_mm

Add param to include multimedia in ccz download

dannyroberts committed:
assert no tabs, CR, LF in ptop reindexer sort_key

dannyroberts committed:
add comment to _load_from_view_and_sort

Commit:
Merge pull request #11991 from dimagi/sort-fast_reindex

Sort fast reindex

esoergel committed:
update deployment submodule

proteusvacuum committed:
Probably

czue committed:
Merge pull request #11998 from dimagi/silly-comment

Remove deprecated comment

czue committed:
Merge pull request #11990 from dimagi/circular-test-import

stop circular import

snopoke committed:
Merge pull request #11994 from dimagi/fix-modal

Fix Edit User Fields modal

czue committed:
fix exports and other celery ajax calls

http://manage.dimagi.com/default.asp?229467

snopoke committed:
Merge pull request #11999 from dimagi/export-fix

fix exports and other celery ajax calls

kaapstorm committed:
Test for "relevant" attr in sync-request/post tag

Commit:
added nested aggregation

Commit:
dimagimon

snopoke committed:
update commcare-hq-deploy

NoahCarnahan committed:
add test

snopoke committed:
exlude system forms from daily form activity

proteusvacuum committed:
Merge pull request #11882 from dimagi/syncrequest_relevant

Use sync-request/post/@relevant to check casedb before claiming case

orangejenny committed:
limit textarea resizing

orangejenny committed:
Merge branch 'master' into limit-resize

orangejenny committed:
Merge pull request #12000 from dimagi/exclude-system-forms

exlude system forms from daily form activity

snopoke committed:
Merge pull request #12002 from dimagi/limit-resize

Limit resizing of textareas

czue committed:
Merge pull request #11987 from dimagi/case-block-fix

Allow floats and other numbers in CaseBlock

Commit:
remove whitespace

snopoke committed:
copy user roles when cloning domain

snopoke committed:
Merge pull request #11828 from lwyszomi/226216-case

226216-case: Unsubscribe from scheduled emails - updated functionality

orangejenny committed:
Trap errors encountered during get_form_data_schema

esoergel committed:
Merge pull request #11992 from dimagi/remove-hashcompat

remove obsolete hash_compat.py

Commit:
Use _rev instead of _id when checking to clear cache

esoergel committed:
Merge pull request #11993 from dimagi/forms.utils

replace django.forms.util with django.forms.utils

Commit:
Add MPR / ASR static data sources for ICDS

czue committed:
Merge pull request #12003 from dimagi/clone

copy user roles when cloning domain

czue committed:
add notify_someone context manager

czue committed:
add feature flag to get notified when a data source rebuild succeeds/fails

NoahCarnahan committed:
Merge pull request #12006 from dimagi/icds-static-ucr

Add MPR / ASR static data sources for ICDS

Commit:
update staging.yaml

[ci skip]

biyeun committed:
update staging.yaml

[ci skip]

biyeun committed:
update staging.yaml

[ci skip]

biyeun committed:
update staging.yaml

[ci skip]

biyeun committed:
update staging.yaml

[ci skip]

biyeun committed:
update staging.yaml

[ci skip]

Commit:
Update submodules

esoergel committed:
Merge pull request #12008 from dimagi/fix-j2me

ensure that endpoints are updated for j2me

Commit:
update staging.yaml

[ci skip]

Commit:
update staging.yaml

[ci skip]

millerdev committed:
Merge pull request #11963 from dimagi/get_app_sucks

Misc cleanup for `get_apps()`

Commit:
Always default sort key to a string

esoergel committed:
Merge pull request #12010 from dimagi/fix-form-reindexer

Always default sort key to a string

Commit:
update staging.yaml

[ci skip]

Commit:
update staging.yaml

[ci skip]

Commit:
remove unused imports from custom/

esoergel committed:
Allow target=save in media_profile.ccpr dl

snopoke committed:
exlude rows with no doc

Commit:
Merge pull request #12019 from dimagi/reindex

exlude rows with no doc

Commit:
Add reindex commands for sql cases and forms

snopoke committed:
Merge pull request #12020 from dimagi/reindex-sql-forms-cases

Add reindex commands for sql cases and forms

Commit:
fix a bunch of tests in Django 1.8

czue committed:
Merge pull request #12005 from dimagi/clear-domain-has-apps-cache

Use _rev instead of _id when checking to clear cache

czue committed:
Merge pull request #11989 from dimagi/fix-time-since

accept 0 as being valid field value

Commit:
Added facets to commcare version report

czue committed:
ensure correct domain used

snopoke committed:
rebuild schema's on error

czue committed:
fix bad variable ref

czue committed:
add entries function

snopoke committed:
exclude system forms from counts by user

snopoke committed:
rename imports

snopoke committed:
exclude usercases from report case counts

czue committed:
make datetime a property

snopoke committed:
filter by case type if supplied

snopoke committed:
exclude user case

czue committed:
[ci skip] lint

snopoke committed:
add error message

snopoke committed:
log exception if exception

snopoke committed:
don't die if no schema

snopoke committed:
Merge pull request #12023 from dimagi/loc-inventory-to-es

minor cleanup and tests to inventory by location report

snopoke committed:
Merge pull request #12004 from dimagi/trap-form-schema-error

Trap errors encountered during get_form_data_schema

czue committed:
fix reporting rate report crashes

http://manage.dimagi.com/default.asp?229623

czue committed:
remove date filter from aggregate inventory report

czue committed:
remove date filter from CurrentStockStatusReport

czue committed:
clean up comments

czue committed:
remove date filter from stock status map report

czue committed:
remove commented out line

czue committed:
cleanup and fix tests

czue committed:
Merge pull request #12022 from dimagi/rebuild-schema

rebuild schema's on error

snopoke committed:
Merge pull request #12025 from dimagi/reporting-rate-crash

fix reporting rate report crashes

czue committed:
Merge pull request #12026 from dimagi/kill-date-filters

Kill date filters on supply reports

NoahCarnahan committed:
submodule update

NoahCarnahan committed:
Merge remote-tracking branch 'origin/master' into prelogin-link

snopoke committed:
update deploy

snopoke committed:
Merge pull request #12031 from dimagi/prelogin-link

Change sign-up link on pricing page to registration page

orangejenny committed:
Merge pull request #12024 from dimagi/system

exclude system forms and 'usercase' from reports

Commit:
Merge pull request #12017 from dimagi/target-save

Allow target=save in media_profile.ccpr dl

Commit:
remove testrunner.py

snopoke committed:
Merge pull request #12007 from dimagi/ucr-notify

add feature flag to get notified when a data source rebuild succeeds/fails

esoergel committed:
Merge pull request #12021 from dimagi/django-18-tests

fix a bunch of tests in Django 1.8

esoergel committed:
Merge pull request #12014 from dimagi/remove-unused-imports-custom

remove unused imports from custom/

emord committed:
Merge pull request #11965 from lwyszomi/hq_version_report

Added Commcare version report

dannyroberts committed:
fix delete_couch_devicelogs memory issue

- remove accidental include_docs=True when trying to get just ids

esoergel committed:
Merge pull request #12033 from dimagi/remove-testrunner.py

remove testrunner.py

dannyroberts committed:
Merge pull request #12037 from dimagi/delete_couch_devicelogs-fix

fix delete_couch_devicelogs memory issue

Commit:
Merge branch 'master' into case-list-icons

dimagimon commented on corehq/apps/app_manager/tests/test_app_manager.py:
```diff
@@ -41,7 +41,7 @@ def setUpClass(cls):
         create_domain(cls.domain)
 
-        with codecs.open(os.path.join(os.path.dirname(__file__), "data", "itext_form.xml"), encoding='utf-8') as f:
+        with codecs.open(os.path.join(os.path.dirname(__file__), "data", "very_simple_form.xml"), encoding='utf-8') as f:
```
E501 line too long (121 > 115 characters)
